consoleClient/main.go

```diff
@@ -0,0 +1,374 @@
+	fmt.Println("9 logout")
+	fmt.Println("10 quit")
+	fmt.Println("====================================================")
+}
+
+func main() {
+	config, err := ReadConfig()
+	if err != nil {
+		log.Fatal(err)
+	}
+	URL = config.ServerAddress
+
+	fmt.Println("====================================================")
+	fmt.Println("Welcome to the smart frige console client")
+	var choice int
+
+	for choice != 10 {
+		printMenu()
+		fmt.Scan(&choice)
+		switch choice {
+		case 1:
+			GetAllRecipes()
+		case 2:
+			if !isAuthorized() {
+				continue
+			}
+			GetAllProducts()
+		case 3:
+			if !isAuthorized() {
+				continue
+			}
+			fmt.Println("Enter produc name")
+			var name string
+			fmt.Scan(&name)
+			GetProductByName(name)
+		case 4:
+			if !isAuthorized() {
+				continue
+			}
+			var name, unit, image string
+			var shelfLife int
+			fmt.Println("Enter produc name")
+			fmt.Scan(&name)
+			fmt.Println("Enter produc shelf life")
+			fmt.Scan(&shelfLife)
+			fmt.Println("Enter produc image link")
+			fmt.Scan(&image)
+			fmt.Println("Enter produc measuring unit")
+			fmt.Scan(&unit)
+			AddProduct(name, shelfLife, image, unit)
+		case 5:
+			if !isAuthorized() {
+				continue
+			}
+			fmt.Println("Enter produc ID")
+			var id string
+			fmt.Scan(&id)
+			DeleteProductByID(id, session)
+		case 6:
+			if !isAuthorized() {
+				continue
+			}
+			fmt.Println("Enter produc ID")
+			var id, name, shelfLife, image, unit string
+			fmt.Scan(&id)
+			p, err := GetProductByID(id)
+			if err != nil {
+				fmt.Println(err)
+				continue
+			}
+
+			fmt.Println("Old product name is " + p.Name + "; Enter new produc name (enter n if you do not wont to change it)")
+			fmt.Scan(&name)
+			if name != "n" {
+				p.Name = name
+			}
+			fmt.Println("Old product shelf life is " + strconv.Itoa(p.ShelfLife) + "; Enter new produc shelf life (enter n if you do not wont to change it)")
+			fmt.Scan(&shelfLife)
+			if shelfLife != "n" {
+				shelfLifeInt, err := strconv.Atoi(shelfLife)
+				if err != nil {
+					fmt.Println(err)
+					continue
+				}
+				p.ShelfLife = shelfLifeInt
+			}
+			fmt.Println("Old image link is " + p.Image + "; Enter new produc image link (enter n if you do not wont to change it)")
+			fmt.Scan(&image)
+			if image != "n" {
+				p.Image = image
+			}
+			fmt.Println("Old product measuring unit is " + p.Units + "; Enter new produc measuring unit (enter n if you do not wont to change it)")
+			fmt.Scan(&unit)
+			if unit != "n" {
+				p.Units = unit
+			}
+			UpdateProduct(p)
+		case 7:
+			if !isAuthorized() {
+				continue
+			}
+			Recepies()
+		case 8:
+			var login, pass string
+			fmt.Println("Enter login")
+			fmt.Scan(&login)
+			fmt.Println("Enter password")
+			fmt.Scan(&pass)
+			loginFunc(login, pass)
+		case 9:
+			if !isAuthorized() {
+				continue
+			}
+			logout()
+			fmt.Println("Logged out")
+		case 10:
+			fmt.Println("Bye!")
+		default:
+			fmt.Printf("Are you sure you have chosen properly?")
+		}
+	}
+}
```

Comment:
You can move your code from the switch by creating a separate function for each case. It will look better.